src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -2979,102 +2979,78 @@ namespace Akka.Event
         public override Akka.Event.LogLevel LogLevel() { }
     }
 }
-namespace Akka.IO
+namespace Akka.IO.Buffers
 {
-    public class ByteBuffer
-    {
-        public ByteBuffer(byte[] array) { }
-        public ByteBuffer(byte[] array, int offset, int length) { }
-        public bool HasRemaining { get; }
-        public int Remaining { get; }
-        public static Akka.IO.ByteBuffer Allocate(int capacity) { }
-        public byte[] Array() { }
-        public void Clear() { }
-        public void Flip() { }
-        public void Get(byte[] ar, int offset, int length) { }
-        public void Get(byte[] ar) { }
-        public void Limit(int maxBufferSize) { }
-        public void Order(Akka.IO.ByteOrder byteOrder) { }
-        public void Put(byte[] src) { }
-        public void Put(byte[] array, int from, int copyLength) { }
-        public void Put(Akka.IO.ByteBuffer src, int length) { }
-        public static Akka.IO.ByteBuffer Wrap(byte[] array, int start, int len) { }
-        public static Akka.IO.ByteBuffer Wrap(byte[] array) { }
+    public class BufferPoolAllocationException : Akka.Actor.AkkaException
+    {
+        public BufferPoolAllocationException(string message) { }
+        public BufferPoolAllocationException(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) { }
+    }
+    public sealed class DefaultBufferPool : Akka.IO.Buffers.IBufferPool
+    {
+        public const int Retries = 30;
+        public DefaultBufferPool(int bufferSize, int buffersPerSegment, int initialSegments, int maxSize) { }
+        public void Release(System.ArraySegment<byte> buf) { }
+        public void Release(System.Collections.Generic.IEnumerable<System.ArraySegment<byte>> buffers) { }
+        public System.ArraySegment<byte> Take() { }
+        public System.Collections.Generic.IEnumerable<System.ArraySegment<byte>> Take(int minimumSize) { }
+    }
+    public interface IBufferPool
+    {
+        void Release(System.ArraySegment<byte> buf);
+        void Release(System.Collections.Generic.IEnumerable<System.ArraySegment<byte>> buf);
+        System.ArraySegment<byte> Take();
+        System.Collections.Generic.IEnumerable<System.ArraySegment<byte>> Take(int minimumSize);
     }
+}
+namespace Akka.IO
+{
     public enum ByteOrder
     {
         BigEndian = 0,
         LittleEndian = 1,
     }
-    public abstract class ByteString : System.Collections.Generic.IEnumerable<byte>, System.Collections.Generic.IReadOnlyCollection<byte>, System.Collections.Generic.IReadOnlyList<byte>, System.Collections.IEnumerable, System.IEquatable<Akka.IO.ByteString>
+    public sealed class ByteString : System.Collections.Generic.IEnumerable<byte>, System.Collections.IEnumerable, System.IEquatable<Akka.IO.ByteString>
```

Comment:
Ok.
What about the rest of the new APIs? Like BufferPoolAllocationException, DefaultBufferPool, IBufferPool, ByteHelpers